spec/factories/posts.rb

```diff
@@ -0,0 +1,5 @@
+FactoryBot.define do
+  factory :post do
+    
```

Comment:
What about declaring the association to user so that FactoryBot creates a user when creating a Post? This way you won't need to create a user yourself to feed it into FactoryBot. 

It's as simple as:

```
FactoryBot.define do
  factory :post do
    message { "MyText" }
    association :author_id
```

Since `author_id` is the alias you've chosen for `user` you'll have to declare this alias in the User factory, and FactoryBot will know what to do.

[OPTIONAL]